src/Maker/MakeAuthenticator.php

```diff
@@ -123,15 +125,9 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
         );
 
         if (self::AUTH_TYPE_FORM_LOGIN === $input->getArgument('authenticator-type')) {
-            $neededDependencies = [TwigBundle::class => 'twig'];
-            $missingPackagesMessage = 'Twig must be installed to display the login form.';
-
-            if (Kernel::VERSION_ID < 40100) {
-                $neededDependencies[Form::class] = 'symfony/form';
-                $missingPackagesMessage = 'Twig and symfony/form must be installed to display the login form';
-            }
+            $neededDependencies = [TwigBundle::class => 'twig', Form::class => 'symfony/form'];
+            $missingPackagesMessage = $this->addDependencies($neededDependencies, 'Twig and symfony/form must be installed to display the login form.');
```

Comment:
I don't remember (and it's a bit old and hard to track down) the reason for the `if (Kernel::VERSION_ID < 40100) {` statement above, but I think you have it backwards. Previously, if the user's version was LESS than 4.1, `symfony/form` was installed. So, if 4.1 or greater, it is NOT required.

Reply:
i cant remember either and grepping for `symfony/form` was - well.. ya.. haha removed, we'll see what the tests say..